examples/let-expression/let_expression.description

```diff
@@ -0,0 +1 @@
+A let expression binds variables and then evaluates an expression with those variables in scope.
```

Comment:
```suggestion
A let expression binds variables and then evaluates an expression with those variables in the scope.
```

Reply:
Fixed.

Reply:
I don't think we need the "the" here. @pubudu91 thoughts?

Please check "in scope" in https://en.wikipedia.org/wiki/Scope_(computer_science)